## 82% of end-end tests have passed
```
Total executed tests: 22
Failed tests: 3
Expected to fail tests: 1
Passed tests: 18
```
```
IDs of failed tests: 702839,702843,741968 
```
```
IDs of expected to fail tests: 702730 
```
<h3>Failed tests (3)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=fix/pinning-message&PR_ID=22553&APK_NAME=22553.apk&TR_CASE_IDS=702839,702843,741968">Rerun failed tests</a></li><br/><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3847545">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><p><blockquote><div>Device 2: Looking for a message by text: delete for me </div><div>Device 1: Looking for a message by text: delete for me </div></blockquote></p>

```
communities/test_community.py:412: in test_community_message_delete
    self.errors.verify_no_errors()
base_test_case.py:200: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Device 1: Deleted message is shown in channel
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESIVSWIJdwIqhWaNjD">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESrrpRWWMA5WJg5XXE">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3847546">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><p><blockquote><div>Device 1: Wait for text element `EmojisNumber` to be equal to `1` </div><div>Device 1: Find `EmojisNumber` by `xpath`: `//*[starts-with(@text,'Message AFTER edit 2 (Edited)')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']/../..//*[@content-desc='emoji-reaction-2']/android.widget.TextView[2]` </div></blockquote></p>

```
communities/test_community.py:384: in test_community_message_edit
    self.errors.verify_no_errors()
base_test_case.py:200: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Device 2: Message is not edited
E    Device 1: Message reaction is not shown for the sender
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESIVSWIJdwIqhWaNjD">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESrrpRWWMA5WJg5XXE">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestProfileMultipleDevices:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3847559">test_profile_change_username</a>, id: 741968 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: LogInButton </div><div>## Signed in successfully!</div></blockquote></p>

```
profile/test_profile.py:180: in test_profile_change_username
    self.errors.verify_no_errors()
base_test_case.py:200: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Device 2: Updated username 'user 1' is not shown in the 1-1 chat
E    Device 2: Updated username is not shown in the user's profile
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESi1POTyCWDxpoRAH7">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES0wPNKLpGsp5c8vZ9">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Expected to fail tests (1)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3847541">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><p><blockquote><div>Device 2: Wait for text element `EmojisNumber` to be equal to `1` </div><div>Device 2: Find `EmojisNumber` by `xpath`: `//*[starts-with(@text,'Message sender')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']/../..//*[@content-desc='emoji-reaction-2']/android.widget.TextView[2]` </div></blockquote></p>

```
private_chats/test_one_to_one_chat.py:65: in test_1_1_chat_message_reaction
    message_receiver.emojis_below_message(emoji="thumbs-up").wait_for_element_text(1, 90)
../views/base_element.py:173: in wait_for_element_text
    element_text = self.find_element().text.strip()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: EmojisNumber by xpath: `//*[starts-with(@text,'Message sender')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']/../..//*[@content-desc='emoji-reaction-2']/android.widget.TextView[2]` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception 
```

<b>[[Might fail from time to time as reaction are set too slowly]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES0S9Zmfs9MNaM6yNU">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESdsQkLkWEyoNd9nQQ">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (18)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3847543">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESXru5WGl1TpSkQPyf">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES0lsDbxQqb8pYSgJi">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESIndnl8Lftxzf90DQ">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDeviceTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3847551">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTEScnVo4L7KEifcDqUc">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3847549">test_send_eth[Status Network Sepolia-0.0002]</a>, id: 727229 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES2HSi8c7mzMu80Yzv">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESgCZbfAXBZI9mrxkp">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3847550">test_wallet_send_erc20_from_drawer[Optimism Sepolia-USD Coin-USDC-2-0.01]</a>, id: 727230 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES2HSi8c7mzMu80Yzv">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESgCZbfAXBZI9mrxkp">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3847561">test_send_eth[Arbitrum Sepolia-0.0001]</a>, id: 742015 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES2HSi8c7mzMu80Yzv">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESgCZbfAXBZI9mrxkp">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3847562">test_wallet_send_erc20_from_drawer[Sepolia-USD Coin-USDC-2-0.01]</a>, id: 742016 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES2HSi8c7mzMu80Yzv">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESgCZbfAXBZI9mrxkp">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestProfileMultipleDevices:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3847560">test_profile_change_profile_photo</a>, id: 741969 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESi1POTyCWDxpoRAH7">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES0wPNKLpGsp5c8vZ9">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3847548">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTES0S9Zmfs9MNaM6yNU">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESdsQkLkWEyoNd9nQQ">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3847544">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESIVSWIJdwIqhWaNjD">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESrrpRWWMA5WJg5XXE">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3847547">test_community_one_image_send_reply_set_reaction</a>, id: 702859 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESIVSWIJdwIqhWaNjD">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESrrpRWWMA5WJg5XXE">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletCollectibles:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3847557">test_wallet_send_collectible</a>, id: 741840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESJVE1AudjR6hjZHif">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3847556">test_wallet_collectibles_balance</a>, id: 741839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESJVE1AudjR6hjZHif">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3847558">test_wallet_collectible_send_from_expanded_info_view</a>, id: 741841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESJVE1AudjR6hjZHif">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3847542">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESLX3H69KfdP5o4zvM">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3847554">test_wallet_swap_flow_mainnet</a>, id: 741555 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESzrZ1EZ72R8Sai1Df">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3847552">test_wallet_balance_mainnet</a>, id: 740490 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESzrZ1EZ72R8Sai1Df">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3847555">test_wallet_bridge_flow_mainnet</a>, id: 741612 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESzrZ1EZ72R8Sai1Df">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3847553">test_wallet_send_flow_mainnet</a>, id: 741554 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://appautomation.lambdatest.com/test?testID=APPTESzrZ1EZ72R8Sai1Df">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/src/status_im/contexts/chat/messenger/menus/pinned_messages/view.cljs
+++ b/src/status_im/contexts/chat/messenger/menus/pinned_messages/view.cljs
@@ -34,8 +34,9 @@
        :description (i18n/label :t/no-pinned-messages-desc)}]]))
 
 (defn view
-  [{:keys [chat-id disable-message-long-press?]}]
+  [{:keys [chat-id]}]
   (let [pinned                 (rf/sub [:chats/pinned-sorted-list chat-id])
+        screen-id              (quo.context/use-screen-id)
         render-data            (rf/sub [:chats/current-chat-message-list-view-context :in-pinned-view])
         current-chat           (rf/sub [:chats/chat-by-id chat-id])
         {:keys [community-id]} current-chat
@@ -61,7 +62,7 @@
      (if (pos? (count pinned))
        [rn/flat-list
         {:data        pinned
-         :render-data (assoc render-data :disable-message-long-press? disable-message-long-press?)
+         :render-data (assoc render-data :disable-message-long-press? (not= screen-id :screen/chat))
          :render-fn   message-render-fn
          :footer      [rn/view {:style style/list-footer}]
          :key-fn      list-key-fn

--- a/src/status_im/contexts/chat/messenger/messages/pin/events.cljs
+++ b/src/status_im/contexts/chat/messenger/messages/pin/events.cljs
@@ -132,7 +132,4 @@
    cofx
    {:padding-bottom-override 21
     :content                 (fn [] [pinned-messages-menu/view
-                                     {:chat-id                     chat-id
-                                      :disable-message-long-press? (not= :chat
-                                                                         (get-in cofx
-                                                                                 [:db :view-id]))}])}))
+                                     {:chat-id chat-id}])}))

